rebasehelper/checkers/pkgdiff_tool.py

```diff
@@ -38,7 +38,7 @@ class PkgDiffTool(BaseChecker):
 
     CMD = "pkgdiff"
     DEFAULT = True
-    pkgdiff_results_filename = 'pkgdiff_reports.html'
+    pkgdiff_results_filename = 'pkgdiff_reports'
```

Comment:
I think this name doesn't make much sense. `pkgdiff` is redundant, and `reports` shouldn't be plural, as it's just one report. Simple `report` sound best to me.